app/indexers/hyrax/admin_set_indexer.rb

```diff
@@ -7,7 +7,7 @@ class AdminSetIndexer < ActiveFedora::IndexingService
     def generate_solr_document
       super.tap do |solr_doc|
         # Makes Admin Sets show under the "Admin Sets" tab
-        Hyrax.config.index_field_mapper.set_field(solr_doc, 'generic_type', 'Admin Set', :facetable)
+        solr_doc['generic_type_sim'] = ['Admin Set']
```

Comment:
How about setting these in a lookup table?  Like `Hyrax.config.solr_fields = { type: 'generic_type_sim' }`?

Reply:
Maybe. I think you might start working your way back to something like solrizer, though, because there will often be more than one solr field for a given model property. I'm advocating to move toward "just let everyone use the solr field they want to use". The dynamic fields are pretty well documented in the solr config. And I think this is a direction the community has been more or less comfortable with for a while.

Reply:
Let's walk through that "just let everyone use the solr field they want to use" scenario.  What if I want to use `primary_type_sim` instead of `generic_type_sim`.  How would I go about that?

Reply:
Write an indexer that adds the field you want. And then either stop using the one you don't want or have your indexer remove it I guess.

Reply:
@jcoyne are you trying to say that this field in particular is uniquely in need of being overridden downstream? Could you flesh out the use case?

Reply:
Okay, so the plan is "write your own (or override the default indexers)". I can live with that.

Reply:
:+1: these are already hard coded in many places, including adjacent to changes in this diff. It seems fine to just normalize on "you customize index keys by changing the indexer to use different keys".

We're going to get something to that effect into the release notes.